src/freertos_drivers/esp_idf/EspIdfWiFi.cxx

```diff
@@ -303,8 +303,13 @@ void EspIdfWiFiBase::mdns_service_add(const char *service, uint16_t port)
     OSMutexLock locker(&lock_);
     if (!mdnsAdvInhibit_)
     {
-        ::mdns_service_add(
+        esp_err_t result = ::mdns_service_add(
             nullptr, name.c_str(), proto.c_str(), port, nullptr, 0);
+        if (result != ESP_OK)
```

Comment:
any reason not to use `ESP_ERROR_CHECK` which captures and logs if it is not ESP_OK?